examples/spin-timer/deps/spin/variables.wit

```diff
@@ -0,0 +1,13 @@
+package fermyon:spin@2.0.0
+
+interface variables {
+    // Get an application variable value for the current component.
+    // The name must match one defined in in the component manifest.
+    get: func(name: string) -> result<string, error>
+
+    variant error {
+        invalid-name(string),
+        provider(string),
```

Comment:
I don't love the name `provider`, but I'm not sure I have a better name. The verbose option (that I'm not really suggesting) would be `provider-specific-error`. Hmm....

Reply:
I don't personally have a strong preference here (i like `provider` but probably just comfortable staring at by now :p) but if you have a concrete suggestion i'm willing to use it!